server/api/repos/[repo_id]/clone.post.ts

```diff
@@ -78,8 +78,7 @@ export default defineEventHandler(async (event) => {
 
     console.log('wrote', issues.length, 'issues');
 
-    // TODO: improve stop condition
-    if (issues.length < 50) {
+    if (issues.length < 50 || page === total) {
```

Comment:
@anbraten yeah you are right. But I think we can also use the totalPages instead of total(I am getting total pages from github and gitlab also supplies it).